src/NLog/LayoutRenderers/DateLayoutRenderer.cs

```diff
@@ -75,8 +75,11 @@ public DateLayoutRenderer()
         [DefaultValue(false)]
         public bool UniversalTime { get; set; }
 
-        private static char[] lowTimeResolutionChars = new char[] { 'Y', 'y', 'M', 'D', 'd', 'H', 'h' };
+        private const string lowTimeResolutionChars = "YyMDdHh";
```

Comment:
Is it really called once? Cannot see that from the code.

Also AFAIK a hashset with 3 items is already more efficient, when used repeatedly